classes/person.rb

```diff
@@ -1,16 +1,16 @@
-require_relative './nameable'
-require_relative './base_decorator'
-require_relative './capitalize_decorator'
-require_relative './trimmer_decorator'
+require_relative 'nameable'
+require_relative 'base_decorator'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..10_000)
-
+    @rentals = []
```

Comment:
- [ ] A method for adding rentals should be present. reference
`def add_rental(book, date)`
`Rental.new(date, book, self)
  end`